WhatsApp/app/src/main/res/values/strings.xml

```diff
@@ -32,4 +57,22 @@
     <string name="changenumber">Change number</string>
     <string name="requestaccountinfo">Request account info</string>
     <string name="deletemyaccount">Delete my account</string>
+
+    <!-- Strings used for notification page in settings area -->
+    <string name="conversationtones">Conversation tones</string>
+    <string name="playsounds">Play sounds for incoming and outgoing\nmessages</string>
```

Comment:
Whatsapp did not use the newline /n approach. They use margin i believe. Would look weird on a horizontal orientation.